### DISCUSSION TITLE
Possibly boards to look at

### DISCUSSION THREAD

**Discussion by ldrrp** [TheGuyDanish/CM4_MATX#55] [Show and tell]
Perhaps some inspiration for ideas or things that may be unseen. But take a look at this. http://linuxgizmos.com/carrier-board-extends-raspberry-pi-compute-module-4-with-pcie-sata/

**Comment by volkertb** [comment]
Perhaps we can learn or copy some things from the NuXT project, since that is also an open source MicroATX motherboard design, which is already being manufactored, sold and used? The hardware and the target applications may be completely different, but perhaps there is still be some stuff in the design, perhaps relating to the form factor, which we could borrow.
